Neos.Media.Browser/Configuration/Settings.yaml

```diff
@@ -27,6 +27,9 @@ Neos:
         # By default, enable the option to create redirects for replaced asset resources
         createAssetRedirectsOption:
           enable: true
+        # By default, enable showing the workspace owner/title in media browser usage tab
+        showWorkspaceOwnerOrName:
```

Comment:
IMO we don't need this switch. If someone doesn't want the feature they can disable or remove the privilege.